docs/contracts/functions.rst

```diff
@@ -383,7 +383,7 @@ operations as long as there is enough gas passed on to it.
 .. warning::
     A ``payable`` fallback function is also executed for
     plain Ether transfers, if no :ref:`receive Ether function <receive-ether-function>`
-    is present. It is recommended to always define a receive Ether
+    is present. It is recommended to always define a received Ether
```

Comment:
This one is not a typo. It's about a function literally named `receive`. If it's unclear, we may be better off rewording and putting it in backticks.